### PR TITLE
Update material-ui.json

### PR DESCRIPTION
**Pull request by oliviertassinari**
Following #207

### CODE REVIEW

File: configs/material-ui.json [hunk 1]
```diff
@@ -47,7 +46,5 @@
     }
   },
   "min_indexed_level": 1,
-  "js_render": true,
```
Comment (oliviertassinari) [inline configs/material-ui.json]: Our docs pages are now server side rendered. This change should speed up the crawling.

File: configs/material-ui.json [hunk 2]
```diff
@@ -18,7 +18,6 @@
   "selectors": {
     "lvl0": {
       "selector": ".algolia-drawer .algolia-lvl0",
-      "type": "xpath",
```
Comment (oliviertassinari) [inline configs/material-ui.json]: Don't miss CSS selector with xpath config.

**Comment by maxiloc** [comment]
I updated a bit the config and deploy. Should be live in a few minutes

**Comment by oliviertassinari** [comment]
It's working like a charm 💃 .

**Pull request merged**